index.html

```diff
@@ -42,13 +44,218 @@ <h1>Organic</h1>
+    </section>   
+
+    <!-------Organic food Section------------>
+
+    <section>
+       <div class="section-organic">
+           
+            <div class="left-col">
+              
+            </div>
+       
+            <div class="right-col">
+                <h3>Organic Farm</h3>
+                <h2>Our Promise</h2>
+                <p>The board game "Farm" is a great option to spend your leisure time in an exciting and 
+                fun way with friends or family. An original plot, a wide variety of strategies, interesting
+                and colorful characters open up wide opportunities for the development of a child’s imagination.</p>
+                <button type="button" class="btn">LEARN MORE</button>
+            </div> 
+            
+            <div class="left-col-1">
+                <h3>Halth Mesefits</h3>
+                <h2>of organic food</h2>
+                <p>The board game "Farm" is a great option to spend your leisure time in an exciting and 
+                fun way with friends or family. An original plot, a wide variety of strategies, interesting
+                and colorful characters open up wide opportunities for the development of a child’s imagination.</p>
+                <button type="button" class="btn">LEARN MORE</button>
+            </div> 
+            
+            <div class="right-col-1">
+                 
+            </div>
+        
+        
+        </div>
+   </section>
+
+
+   <!--------About our Farm Section------------>
+
+<section>
+    <div class="about-farm">
+        <h2>About Our Farm</h2>
+        <p>Give each player a personal tablet - this is a farm plot. 
+        In addition to the pen in which you will keep, 
+        breed and exchange your animals, it shows tips - 
+        the exchange rate between your pets.</p>
+
+        <div class="box-about">
+            <div class="about">
+                <h3>Vegetables</h3>
+                <p>Give each player a personal tablet - this is a farm 
+                plot.</p>
+                <img src="img/vegetables-hands.jpg" style="width: 275px; 
+                height: 300px;">
+            </div>
+        
+          <div class="about">
+               <h3>How it work</h3>
+               <p>Give each player a personal tablet - this is a farm 
+               plot.</p>
+               <img src="img/greenhouse.jpg" style="width: 275px; 
+               height: 300px;">
+        </div>
+        
+        <div class="about">
+              <h3>Flexibility</h3>
+              <p>Give each player a personal tablet - this is a farm 
+              plot.</p>
+              <img src="img/lettuce.jpg" style="width: 275px; 
+              height: 300px;">
+        </div>
+        
+        <div class="about">
+            <h3>Farm Products</h3>
+            <p>Give each player a personal tablet - this is a farm 
+             plot.</p>
+            <img src="img/apples.jpg" style="width: 275px; 
+              height: 300px;">
```

Comment:
If attributes doesn't fit into one line, then every of them should start on a new line like this:
<img
    src="img/apples.jpg"
    style="width: 275px; height: 300px;"
 />
 
 
But anyway try to avoid inline styles in the future and use css classes instead...